components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -394,6 +394,7 @@ be lost if SCREAM_HACK_XML is not enabled.
     <energy_column_conservation_error_tolerance>1e-14</energy_column_conservation_error_tolerance>
     <column_conservation_checks_fail_handling_type>Warning</column_conservation_checks_fail_handling_type>
     <check_all_computed_fields_for_nans type="logical">true</check_all_computed_fields_for_nans >
+    <property_check_data_fields type="array(string)" doc="list of additional data fields to output in property checks">phis,landfrac</property_check_data_fields>
```

Comment:
I'm assuming phis and landfrac are always available in CIME runs, right?

Reply:
`phis` is an output field in homme and input in shoc. `landfrac` is imported in the surface coupler. I think those should be around in all cases?

Reply:
Ok, good, just double checking.